tests/NHSD.BuyingCatalogue.Organisations.Api.UnitTests/Services/CreateBuyerServiceTests.cs

```diff
@@ -5,6 +5,7 @@
 using FluentAssertions;
 using Moq;
 using NHSD.BuyingCatalogue.Organisations.Api.Models;
+using NHSD.BuyingCatalogue.Organisations.Api.Models.Results;
 using NHSD.BuyingCatalogue.Organisations.Api.Repositories;
 using NHSD.BuyingCatalogue.Organisations.Api.Services;
 using NHSD.BuyingCatalogue.Organisations.Api.UnitTests.Builders;
```

Comment:
I think we may have a `using` that's not required.

Reply:
👍